src/stories/Typography.stories.tsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
+
+import kebabCase from 'lodash/fp/kebabCase';
+
+/*
+ * TODO: #8709 - Automate documentation of styling fundamentals
+ *
+ * The token objects and React component tables are a pre-cursor to fully
+ * automating the process of documenting the design system. This will go hand
+ * in hand with design token discovery work into using Style Dictionary, which
+ * is underway but not yet completed.
+ *
+ * @see {@link https://github.com/wellcometrust/corporate/issues/8709}
+ */
+export const tokensFontFamilies: NestedTokenProps = {
+  FontPrimary: {
+    value: `'Helvetica Neue', Helvetica, Arial, sans-serif`,
+    comment: 'Standard body text, headings and links'
+  },
+  FontPrimaryAlt: {
+    value: 'Helvetica, Arial, sans-serif',
+    comment:
+      'Alternative for FontPrimary, this font stack removes odd line-height issue encountered with Helvetica Neue'
+  },
+  FontSecondary: {
+    value: `'Wellcome-Bold', sans-serif`,
+    comment: 'Main page headings and quote text (to be used sparingly)'
+  },
+  FontTertiary: {
+    value: `'Courier New', Courier, monospace`,
+    comment: 'Image and video captions, licensing info'
+  }
+};
+
+export const tokensFontSizeHeadings: NestedTokenProps = {
+  FontSizeHeadingXxl: {
+    comment: 'H0 Heading XXL',
+    mqBase: '32',
+    mqSmall: '34',
+    mqMedium: '40'
+  },
+  FontSizeHeadingXl: {
+    comment: 'H1 Heading XL',
+    mqBase: '24',
+    mqSmall: '28',
+    mqMedium: '32'
+  },
+  FontSizeHeadingLg: {
+    comment: 'H2 Heading LG',
+    mqBase: '22',
+    mqSmall: '22',
+    mqMedium: '24'
+  },
+  FontSizeHeadingMd: {
+    comment: 'H3 Heading MD',
+    mqBase: '18',
+    mqSmall: '18',
+    mqMedium: '20'
+  },
+  FontSizeHeadingSm: {
+    comment: 'H4 Heading SM',
+    mqBase: '16',
+    mqSmall: '16',
+    mqMedium: '16'
+  },
+  FontSizeHeadingXs: {
+    comment: 'H5 Heading XS',
+    mqBase: '14',
+    mqSmall: '14',
+    mqMedium: '14'
+  }
+};
+
+export const tokensFontSizeBody: NestedTokenProps = {
+  FontSizeBodyXl: {
+    comment: 'Body XL',
+    mqBase: '24'
+  },
+  FontSizeBodyLg: {
+    comment: 'Body LG',
+    mqBase: '20'
+  },
+  FontSizeBodyMd: {
+    comment: 'Body MD',
+    mqBase: '16'
+  },
+  FontSizeBodySm: {
+    comment: 'Body SM',
+    mqBase: '14'
+  },
+  FontSizeBodyXs: {
+    comment: 'Body XS',
+    mqBase: '12'
+  }
+};
+
+export const tokensTypography: MappedTokensProps = {
+  FontSizeBase: '1rem',
+  FontSizeBasePx: '16',
+  FontWeightHeading: '500',
+  LetterSpacingBody: '0.4px',
+  LetterSpacingHeadingMajor: '-0.5px',
+  LetterSpacingHeadingMinor: '0.2px',
+  LineHeightBody: '1.6',
+  LineHeightHeading: '1.6'
+};
```

Comment:
Same comment as https://github.com/wellcometrust/design-system/pull/9/files#r684183748.

Might be more straightforward/easier-to-move-later if all of these values were defined in a JSON file, probably in the root of this repo?